Alignment/OfflineValidation/python/TkAlAllInOneTool/dataset.py

```diff
@@ -820,15 +826,60 @@ def dump_cff( self, outName = None, jsonPath = None, begin = None,
         theFile.close()
         return
 
-    def fileList( self, parent = False ):
+    def createdatasetfile_hippy(self, filename, filesperjob, firstrun, lastrun):
+        with open(filename, "w") as f:
+            for job in self.__chunks(self.fileList(firstRun=firstrun, lastRun=lastrun, forcerunselection=True), filesperjob):
+                f.write(",".join("'{}'".format(file) for file in job)+"\n")
+
+    @staticmethod
+    def getrunnumberfromfilename(filename):
+        parts = filename.split("/")
+        result = error = None
+        if parts[0] != "" or parts[1] != "store":
```

Comment:
I don't think so... this way catches "something/store" and also "/something", `and` would not.